app/controllers/content_items_controller.rb

```diff
@@ -69,9 +69,13 @@ def load_taxonomy_navigation
 
       taxon_ids = taxons.map { |taxon| taxon["content_id"] }
       services = Supergroups::Services.new(taxon_ids)
+      news_and_communications = Supergroups::NewsAndCommunications.new(taxon_ids)
+      transparency = Supergroups::Transparency.new(taxon_ids)
 
       @taxonomy_navigation = {
         services: (services.all_services if services.any_services?),
+        news_and_communications: news_and_communications.tagged_content,
```

Comment:
We need a method here (and for transparency) that's similar to the one above for services, that first checks to see if there are any services before adding them to the hash.